script.js

```diff
@@ -1,39 +1,97 @@
-let currencyRate = {
-  timestamp: 1519296206, 
-  base: "EUR",
-  date: "2023-12-12", 
-  rates: {},
-};
+let currencyRate = [
+  {
+    timestamp: new Date().toLocaleDateString(),
+    base: "EUR",
+    date: "2023-12-12",
+    rates: {},
+  },
+  {
+    timestamp: new Date().toLocaleDateString(),
+    base: "USD",
+    date: "2023-12-12",
+    rates: [],
+  },
+  {
+    timestamp: new Date().toLocaleDateString(),
+    base: "DKK",
+    date: "2023-12-12",
+    rates: {},
+  },
+];
 
+const specials = {
```

Comment:
you are not using it, if you do not work with special offers, you can delete it :)